ebean-test/src/test/resources/logback-test.xml

```diff
@@ -78,6 +78,7 @@
   <logger name="io.ebean.docker" level="DEBUG"/>
   <logger name="io.ebean.test" level="TRACE"/>
   <logger name="io.ebean.MarkedAsDeleted" level="DEBUG"/>
+  <logger name="io.ebean.test.loggerTest" level="DEBUG"/>
```

Comment:
we can remove this debug here. It was just verify that the console output matches with the assertion